max_rng: 8.0                              # lidar max range [m]

th_r: 60                                  # lidar data processing range [deg]

p_gain: 15                                # control p gain

tread: 0.149                              # robot tread [m]
vel: 0.07                                 # default velocity [m/s]
